src/main/java/org/example/DiagonalMatrix.java

```diff
@@ -2,83 +2,30 @@
 
 import java.util.Scanner;
 
-import static java.lang.Math.abs;
+import static org.example.DoWhileLoopMatrixGenerator.generateMatrixWithDoWhileLoop;
+import static org.example.ForLoopMatrixGenerator.generateMatrixWithForLoop;
+import static org.example.WhileLoopMatrixGenerator.generateMatrixWithWhileLoop;
 
 public class DiagonalMatrix {
     public static void main(String[] args) {
-        // Create a Scanner object to read input from the console
         Scanner scanner = new Scanner(System.in);
-
-        // Prompt the user to enter the value of 'n'
         System.out.print("Enter the value of 'n' (not greater than 100): ");
         int n = scanner.nextInt();
-                // Check if n is greater than 100
+
         if (n > 100) {
             System.out.println("Please enter a value less than or equal to 100.");
             return;
         }
```

Comment:
you don't give the user a second chance to input a valid value. If user inputs a value bigger than 100 your app is terminated. Instead you have to ask for a good value in a loop until the user inputs something valid

Reply:
I have added a while loop instead of if